internal/mocks/translation/audit.go

```diff
@@ -0,0 +1,19 @@
+package translation
```

Comment:
In other PR (or PRs) I put thin under the fake folder.

We should be consistent, so lets ask for others votes here:
@s-urbaniak &  @roothorp WDYT?

`internal/resource/fake`
OR
`internal/mocks/translation` (all mocks in a single package it seems)

Reply:
this is a nuance, leaving the final call to you @josvazg and @helderjs